src/pages/Components/Toggle/previews/toggle-states.preview.html

```diff
@@ -0,0 +1,18 @@
+<ul class="example">
```

Comment:
not required, I will use the same default preview as the one in code, you can delete this